Driverless AI/get-started-with-open-source-custom-recipes-tutorial/get-started-with-open-source-custom-recipes-tutorial.md

```diff
@@ -522,30 +541,30 @@ We are going to get the URL for **Brier Loss** Scorer we used in Task 4 and uplo
 5\. The URL for this version of the recipe should be the one below. This URL is then pasted into Driverless AI.
 
 ~~~html
-https://github.com/h2oai/driverlessai-recipes/blob/rel-1.8.7/scorers/classification/binary/brier_loss.py
+https://github.com/h2oai/driverlessai-recipes/blob/rel-1.9.0/scorers/classification/binary/brier_loss.py
 ~~~
 
 6\. See task 4 of this tutorial to follow the process of uploading a recipe via the raw URL
 
 ### Uploading Recipes from Local Machine 
 
-For this tutorial, we are using **Driverless AI 1.8.7.1,** but we will be working off the Driverless AI 1.8.7 branch, as both, 1.8.7 and 1.8.7.1 branches have been merged.
+For this tutorial, we are using **Driverless AI 1.9.0,** but we will be working off the Driverless AI 1.9.0 branch.
```

Comment:
This note can be deleted. It was only applicable to the 1.8.7.1 release since there were 2 branches merged for recipes.